package.json

```diff
@@ -26,21 +26,20 @@
   "author": "MultiversX",
   "license": "GPL-3.0-or-later",
   "devDependencies": {
-    "@babel/core": "^7.17.8",
-    "@babel/preset-env": "^7.16.11",
-    "@babel/preset-typescript": "^7.16.7",
-    "@types/axios": "^0.14.0",
-    "@types/jest": "^27.4.1",
+    "@babel/core": "^7.23.6",
+    "@babel/preset-env": "^7.23.6",
+    "@babel/preset-typescript": "^7.23.3",
+    "@types/jest": "^29.5.11",
     "@types/node": "^16.11.10",
     "@typescript-eslint/eslint-plugin": "^5.12.0",
     "@typescript-eslint/parser": "^5.16.0",
     "axios-mock-adapter": "^1.20.0",
     "babel-jest": "^27.5.1",
     "eslint": "^8.9.0",
-    "jest": "^27.5.1",
+    "jest": "^29.7.0",
     "typescript": "^4.3.5"
   },
   "dependencies": {
-    "axios": "0.24.0"
+    "axios": "^1.6.2"
   }
-}
+}
```

Comment:
optional: empty line at the end of the file

Reply:
we need to set up linting rules for that and then it will be fixed "automagically". we can make a new PR for this